handler/status.go

```diff
@@ -104,6 +105,10 @@ func Status(
 				"goVersion": version.GoVersion,
 			}
 
+			if pathPrefix != "" && !strings.HasSuffix(pathPrefix, "/") {
+				pathPrefix = pathPrefix + "/"
+			}
+
```

Comment:
Would it also work to just do the below and not change `template.html` at all?
```Go
	if pathPrefix == "" {
		pathPrefix = "."
	}
```

Reply:
Oh that also should work. Nice catch! 
Will updated PR